test/Microsoft.AspNetCore.Server.KestrelTests/ChunkedRequestTests.cs

```diff
@@ -78,6 +78,8 @@ public static TheoryData<ServiceContext> ConnectionFilterData
                          "");
                     await connection.ReceiveEnd(
                         "HTTP/1.1 200 OK",
+                        "Date: " + testContext.DateHeaderValue,
+                        "Server: Kestrel",
```

Comment:
Is it worth setting `AddServerHeader = false` in `TestServiceContext` to get rid of all of these `Server` headers in the tests?